packages/sites/src/_ensure-telemetry.ts

```diff
@@ -26,6 +33,7 @@ export function _ensureTelemetry(model: IModel) {
     }
   };
   deleteProp(clone, "data.values.gacode");
```

Comment:
Do we want to delete this line?

Reply:
I think it's okay to leave it. It will remove that prop from both the draft and the site, so no problem.